src/install.ps1

```diff
@@ -1,15 +1,16 @@
 $ErrorActionPreference = 'Stop'
 $ProgressPreference = 'SilentlyContinue'
 
-$PwrPath = if ($env:PwrHome) { $env:PwrHome } else { "$env:appdata\pwr" }
+$PwrPath = if ($env:PwrHome) { $env:PwrHome } else { "$env:AppData\pwr" }
 $PwrCmd = "$PwrPath\cmd"
+$PwrScriptPath = "$PwrCmd\pwr.ps1"
 mkdir $PwrCmd -Force | Out-Null
-Invoke-WebRequest -UseBasicParsing 'https://raw.githubusercontent.com/airpwr/airpwr/main/src/pwr.ps1' -OutFile "$PwrCmd\pwr.ps1"
+& "$env:SYSTEMROOT\System32\curl.exe" -s --url 'https://raw.githubusercontent.com/airpwr/airpwr/main/src/pwr.ps1' --output $PwrScriptPath
 $UserPath = [Environment]::GetEnvironmentVariable('Path', 'User')
 if (-not $UserPath.Contains($PwrCmd)) {
 	[Environment]::SetEnvironmentVariable('Path', "$UserPath;$PwrCmd", 'User')
 }
 if (-not ${env:Path}.Contains($PwrCmd)) {
 	$env:Path = "$env:Path;$PwrCmd"
 }
-& "$PwrCmd\pwr.ps1" version
\ No newline at end of file
+& $PwrScriptPath version
```

Comment:
One issue is that the "pwr: " part of this command is not color coded. Not sure if we want to try to fix that since you only ever see it once.